### ISSUE TITLE
Not related to x64/x96 dbg

### ISSUE THREAD

**Issue by MechDR** [x64dbg/x64dbg#2251]
I just don't know where to reach mrexodia except here, so here goes nothing...

I've been having an issue with a driver for a scanner which is compiled in MSVC++ 2003 (yes, the very first x64 MS compiler) and well... it's kind of a long story... it didn't work under Windows 7/2008 (BSOD) but it DID work under Windows 8.1/2012 R2 (I knew 8.1/2012 R2 had better compatibility with drivers from earlier experience I've had with the OSes), but... recent updates (from about 2, 3 years ago) might have broken something.

So, basically, it works, but... there's a lag... like 10 or more seconds before anything is responsive.

Anyhow, I'd really like to pick your brain about this thing since... I really don't know where I'm supposed to start with this... I thought about RCEing the heck out of the DLL, but, as I said, I really have no idea where to start :(.

**Comment by mrexodia** [comment]
I have absolutely no clue, sorry :slightly_smiling_face:
